README.md

```diff
@@ -1,2 +1,5 @@
-# stepzen-login
-github action for logging into a stepzen server
+This GitHub Action is for logging into the StepZen Server in the specified `domain`, which defaults to `stepzen.net`. 
```

Comment:
follow on but should follow style of other actions. e.g see https://github.com/actions/checkout